plugin/src/cli_utils/clusterinfo/mod.rs

```diff
@@ -0,0 +1,42 @@
+use crate::cli_utils::CliArgs;
+use plugin::ExecuteOperation;
+use snafu::Snafu;
+
+/// Cluster-info operations.
+#[derive(clap::Subcommand, Debug)]
+pub enum Operations {
+    /// Gets information of all installed engines.
+    Get(ClusterInfoArg),
+}
+
+#[async_trait::async_trait(?Send)]
+impl ExecuteOperation for Operations {
+    type Args = CliArgs;
+    type Error = Error;
+
+    async fn execute(&self, _cli_args: &CliArgs) -> Result<(), Error> {
+        match self {
+            Operations::Get(_cluster_info_arg) => {
+                let _ = dummy_construct();
```

Comment:
what is this doing?

Reply:
That just constructs and returns error. We have defined error but without implementation they were not getting constructed.

Reply:
that's not needed, just need to add #[allow(unused)]